src/simulate.js

```diff
@@ -59,7 +59,7 @@ var launchServer = function (opts) {
         appUrl = urlRoot + parseStartPage(projectRoot);
         simHostUrl = urlRoot + 'simulator/index.html';
         log.log('Server started:\n- App running at: ' + appUrl + '\n- Sim host running at: ' + simHostUrl);
-        return { appUrl: appUrl, simHostUrl: simHostUrl };
+        return { urlRoot: urlRoot, appUrl: appUrl, simHostUrl: simHostUrl };
```

Comment:
I think they both make sense (`urlRoot` = "the root of all URLs served by the server" versus `rootUrl` = "the URL pointing to the root of the server"), but since Tim had already used `urlRoot`, I'm inclined to just leave it like this :) Is that fine?